strategies/RandomStrategy.py

```diff
@@ -0,0 +1,13 @@
+from utils.BaseStrategy import BaseStrategy
+import random
+
+"""
+Strategy:
+    Random Strategy for going Left or Right
+"""
+class RandomStrategy(BaseStrategy):
+    def calculate(self, observation):
+        if random.random():
```

Comment:
shouldn't this be `random.random() < 0.5`?

Reply:
Also I believe this is already implemented on master

Reply:
I think `random.random()` returns a boolean.